runtime/src/bank.rs

```diff
@@ -7001,7 +6923,11 @@ pub(crate) mod tests {
         // not being eagerly-collected for exact rewards calculation
         bank.restore_old_behavior_for_fragile_tests();
 
-        assert_eq!(bank.capitalization(), 42 * 1_000_000_000);
+        let sysvar_and_native_proram_delta = 10;
```

Comment:
there are bunch of similar hard coded magic numbers in this pr; fyi, I'll soon replace them once I create the next and final pr to change the rent calculation for sysvars.

Reply:
... so, please bear with this for now. ;)